language-design.md

```diff
@@ -106,7 +162,23 @@ you do so? If not, why not?
 
 **Response**
 
-
+Not necessarily; Quorum is more structured than AppleScript, so it's possible to include
+natural language, and thus make the language more understandable to novice users, as long
+as the language is structured enough for the user to parse it as proper code. Where AppleScript
+fails is in making the syntax too similar to English, but the actual parsing doesn't support
+syntax that would be acceptable in English, and may be written by the users,
+but isn't handled by the language.
+Quorum is structured enough for its users to understand that their code must be structured,
+so while it's easier for them to understand there is less of a chance they will make syntax errors.
+
+Whether or not to use natural language depends on many factors, such as the purpose of the language and who its users will be, and 
+what parsing libraries are available for natural language syntax. For example, 
+"Quorum... started as an interpreted language originally designed to be 
+easier to hear through screen readers for blind or visually impaired users" [Stefik _et al_., 2014](http://quorumlanguage.com/about.php),
+so using natural language was important as it would be easier to hear.
+Neither of us are particularly opposed to using natural language in a DLS's design, 
```

Comment:
Echoing Alex's point, I would encourage you to think about natural language's ambiguity before deciding to try to make your DSL handle natural language.